launch/sensors_launch.py

```diff
@@ -28,8 +28,9 @@ def generate_launch_description():
     SIMULATION = os.getenv('SIMULATION')
     simulation_mode = (SIMULATION == "1")
 
-    DRONE_TYPE = os.getenv('DRONE_TYPE')
-    is_robot_holybro_type = (DRONE_TYPE == "HOLYBRO")
+    # DRONE_AIRFRAME = <t-drone | holybro>
+    DRONE_AIRFRAME = os.getenv('DRONE_AIRFRAME')
+    is_robot_holybro_type = (DRONE_AIRFRAME == "holybro")
```

Comment:
This works as is, but just a note about forward compatibility, if we ever add third airframe, all non-holybros will be treated as t-drone. If used defensive programming, we could accept ["", "t-drone"] as t-drone, "holybro" as holybro and in all other cases error out telling that we got unrecognized value

But this is acceptable as-is :+1: